avatica-shaded/pom.xml

```diff
@@ -149,5 +153,10 @@ language governing permissions and limitations under the License. -->
             <artifactId>commons-codec</artifactId>
             <version>${commons.codec.version}</version>
         </dependency>
+        <dependency>
+            <groupId>junit</groupId>
```

Comment:
Can anyone tell me why this isn't coming from the parent?  Without it I get: 
```maven
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:3.0.0-M4:test (default-test) on project kafka-connect-storage-common-htrace-core4-shaded: groups/excludedGroups require TestNG, JUnit48+ or JUnit 5 (a specific engine required on classpath) on project test classpath -> [Help 1]
```

Reply:
Not sure why it's not building in your setup. I can build the project locally with `mvn clean package` or `mvn clean verify` from the top level directory of any module of the project.

Reply:
`mvn clean verify worked` for me too, seems like some local issue on Chris's end

```
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:07 min
[INFO] Finished at: 2021-11-18T12:58:44-06:00
[INFO] ------------------------------------------------------------------------
➜  kafka-connect-storage-common git:(master) 
```